src/main/java/sirius/biz/storage/layer1/S3ObjectStorageSpace.java

```diff
@@ -73,7 +81,7 @@ private ObjectStore resolveObjectStore(Extension extension) {
     }
 
     private BucketName bucketName() {
-        return store.getBucketName(name);
+        return store.getBucketName(bucketName);
```

Comment:
This still adds the suffix to the bucket name, wasn't the intention of this change to also supress the suffix?